resilience4j-feign/src/main/java/io/github/resilience4j/feign/FeignDecorators.java

```diff
@@ -0,0 +1,70 @@
+/*
+ *
+ * Copyright 2018
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ *
+ */
+package io.github.resilience4j.feign;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import io.github.resilience4j.circuitbreaker.CircuitBreaker;
+import io.github.resilience4j.ratelimiter.RateLimiter;
+import io.vavr.CheckedFunction1;
+
+/**
+ * Builder to help build stacked decorators.
+ */
+public class FeignDecorators implements FeignDecorator {
```

Comment:
Maybe we can add here that the order of the `withXXX` builder methods als controls the order of the decorators.

Reply:
Yes, I have been thinking about the ordering of the decorators. The question is; would it be better to have a static ordering? For example, does it make sense to have the Ratelimiter before the CircuitBreaker a.s.o.?

Reply:
I think the flexibility should be given